src/native/minipal/cpufeatures.c

```diff
@@ -263,13 +263,6 @@ int minipal_getcpufeatures(void)
                                         {
                                             result |= XArchIntrinsicConstants_Evex;
                                             result |= XArchIntrinsicConstants_Avx10v1;
-
-                                            bool isV512Supported = (cpuidInfo[CPUID_EBX] & (1 << 18)) != 0;     // Avx10/V512
```

Comment:
Should we assert that the `Avx10/V512` bit is true iff `XArchIntrinsicConstants_Avx512` is set?

Reply:
We are relying on this [part](https://github.com/dotnet/runtime/pull/103144/files#diff-eba17a72b6caa8b52b196f3b84bfd20dfeaf39a89f410943a692d1f33f569f3bR1437) to set `Avx10/V512`, now `Avx10/V512` will be set iff `EVEX`, `Avx10v1` and `Avx512` all present.

Reply:
Right, I meant to document this via an assert. Left a suggestion.

Reply:
Thanks for the suggestion!